app/model/YouTubeMessage.scala

```diff
@@ -0,0 +1,28 @@
+package model
+
+import net.logstash.logback.marker.Markers
+import org.joda.time.DateTime
+import play.api.Logger
+
+import scala.collection.JavaConverters._
+
+case class YouTubeMessage(atomId: String, videoId: String, reason: String, message: String, isError: Boolean = false) {
+
+  def logMessage =
```

Comment:
Nitpick, I'd define this `def logMessage()` so it looks like a side-effecty function rather than a getter